src/patterns/webform/examples/errors/index.njk

```diff
@@ -0,0 +1,238 @@
+                    "classes": 'js-inpagelink'
+                }
+            ]
+        })
+    }}
+    
+    {% endcall %}
+    
+    <h1 class="u-fs-xxl u-mt-l">Web form</h1>
+
+    {{ onsRadios({
+        "legend": 'Your country',
+        "name": 'webform-country',
+        "radios": [
+            {
+                "id": 'england',
+                "label": {
+                    "text": 'England'
+                },
+                "value": 'england'
+            },
+            {
+                "id": 'wales',
+                "label": {
+                    "text": 'Wales'
+                },
+                "value": 'wales'
+            },
+            {
+                "id": 'northern-ireland',
+                "label": {
+                    "text": 'Northern Ireland'
+                },
+                "value": 'northern-ireland'
+            }
+        ],
+        "error": {
+            "id": 'webform-country-error',
+            "text": 'Select your country',
+            "dsExample": isPatternLib
+        }
+    }) }}
+
+    {{ onsRadios({
+        "legend": 'What do you need help with?',
+        "name": 'webform-help',
+        "radios": [
+            {
+                "id": 'missing-information',
+                "label": {
+                    "text": 'I don’t have everything I need to complete my census'
+                },
+                "value": 'missing-information'
+            },
+            {
+                "id": 'technical-difficulties',
+                "label": {
+                    "text": 'I am having technical difficulties'
+                },
+                "value": 'technical-difficulties'
+            },
+            {
+                "id": 'need-help',
+                "label": {
+                    "text": 'I need help with my census form'
+                },
+                "value": 'need-help'
+            },
+            {
+                "id": 'complaint',
+                "label": {
+                    "text": 'I have a complaint'
+                },
+                "value": 'complaint'
+            },
+            {
+                "id": 'address-help',
+                "label": {
+                    "text": 'I need help with my address'
+                },
+                "value": 'address-help'
+            },
+            {
+                "id": 'something-else',
+                "label": {
+                    "text": 'Something else'
+                },
+                "value": 'something-else'
+            }
+        ],
+        "error": {
+            "id": 'webform-help-error',
+            "text": 'Select what you need help with',
+            "dsExample": isPatternLib
+        }
+    }) }}
+
+    {{ onsTextarea({
+        "name": 'webform-detail',
+        "id": 'webform-detail',
+        "label": {
+            "text": 'Please provide more detail'
+        },
+        "charCheckLimit": {
+            "limit": 5000,
```

Comment:
I know this was the character limit on census, but I don't understand why it's so high for a web form.  I'd say 2000 is more than enough.

Reply:
Agreed. I took this value from what was used in Census2021. Currently live.